lib/entity.js

```diff
@@ -121,20 +187,12 @@ Entity.prototype.trimSnapshot = function trimSnapshot (snapshot) {
   return snapshot;
 };
 
-Entity.digestMethod = function (type, fn) {
-  if ( ! type) throw new EntityError('type is required for digest method definitions');
-  if ( ! fn) throw new EntityError('a function is required for digest method definitions');
-  if ( ! fn.name) throw new EntityError('Anonmyous functions are not allowed in digest method definitions. Please provide a function name');
-  type.prototype[fn.name] = function () {
-    var digestArgs = Array.prototype.slice.call(arguments);
-    digestArgs.unshift(fn.name);
-    Entity.prototype.digest.apply(this, digestArgs);
-
-    var methodArgs = Array.prototype.slice.call(arguments);
-    return fn.apply(this, methodArgs);
-  };
-};
-
+/**
+ * Remove the internal sourced properties from the passed snapshot.
+ *
+ * [CONFIRM] based on https://github.com/mateodelnorte/sourced/search?utf8=%E2%9C%93&q=digestMethod
```

Comment:
digestMethod is a helper function to automatically create a method that calls digest on the param provided. it cuts down on a small amount of code required to write entity methods, and sets a little bit of convention for how to do so. 

In a digestMethod like: 

```
Entity.digestMethod(Car, function clearSettings (param) {

  const self = this;

  this.settings.get(param.name).forEach((name, config) => {

    config.sources.forEach((source) => {

      source.remove();

    });

  });

  return this.settings;

});
```

an anonymous function gets created that automatically calls digest(param); and executes the clearSettings method on the Entity. Entity.digestMethod works so that it creates the wrapper function and names it with the name of the function you provide, and puts it on the Car (in this case) prototype.